### DISCUSSION TITLE
Packaging Flutter apps for Void Linux !

### DISCUSSION THREAD

**Discussion by hesam-init** [void-linux/void-packages#51276] [Q&A]
Hey everyone, 

I'm trying to package Hiddify Next, a Flutter-based v2ray client, for Void Linux, but I'm a bit lost. Since we can't use prebuilt .deb packages, I need to find the right way to do this. As I've been working on it, I started thinking that we should have a standardized Flutter build style for Void packages.

[Hiddify Next](https://github.com/hiddify/hiddify-next)

**Comment by classabbyamp** [comment]
flutter isn't even packaged so you'd need to do thar before adding any package that needs it or creating a build style

**Reply by hesam-init** [reply]
> flutter isn't even packaged so you'd need to do thar before adding any package that needs it or creating a build style

Before using Flutter, we should add the Dart SDK. Today, I tried to turn it into a package, and it was like a nightmare for me.

Here is the [Dart SDK template](https://github.com/void-land/void-repository/blob/main/srcpkgs/dart-sdk/template).

I think the best way is to use prebuilt packages, but it's against the policies.
